test/Npgsql.Tests/Types/CompositeHandlerTests.Read.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using NUnit.Framework;
+
+namespace Npgsql.Tests.Types
+{
+    public partial class CompositeHandlerTests
```

Comment:
I still prefer on class instead of partial, but if you feel strongly about it I'm OK with this too.

Reply:
There will be more tests. Will add them via #2561.

By the way, Microsoft uses partial classes and I find this very useful. The name of the class says what you're testing, while tests split by areas.

Reply:
That really depends on the number of tests, which doesn't seem very big here. Also, if you have enough tests, I think it makes more sense to split to different classes rather than use partial classes - this allows you to filter by class on the command line, in the IDE, etc...